ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1465,7 +1553,12 @@ connected to your 'admin' network. Use 'none' if you have no gateway.\n\n"
 
 	# Bring the admin and external nics up now: they need to be for the
 	# connectivity checks in the next section
+	printf "\nBringing up the network interfaces.."
+	if [[ $nicsup_done -eq 1 ]]; then
```

Comment:
Why nicsdown or why the message?
1. The nicsdown is neccessary. Otherwise you will get ugly errors when re-running config.
2. The nicsup function takes some time. And some people tend to test if the terminal is working by hitting the enter. But this input is immediately taken by the next question. As a result, the default value is selected without prompting. That's why we need to inform the people about what are we going to do.